storybook/src/components/Paragraph/Paragraph.docs.mdx

```diff
@@ -17,10 +17,11 @@ import README from "../../../../packages/css/src/components/paragraph/README.md?
 ### Large text
 
 Use this size only for the lead paragraph of a page.
-It is not mandatory to use a lead paragraph, but there must be no more than 1 per page.
+A lead paragraph is optional, but there must be no more than one per page.
```

Comment:
[The Schrijfwijzer Amsterdam](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/tekstonderdelen-heldere-taal/getallen-bedragen-breuken-percentages/) prescribes using '1' here, instead of 'one'. Should we adhere to that?